tests/css_less_docs/less_docs.css

```diff
@@ -1076,7 +1076,7 @@ div {
   .mixin();
 }
 
-@size: globaly-defined-value; // callers parent scope - no protection
+@size: globally-defined-value; // callers parent scope - no protection
```

Comment:
Is there a snapshot for this file?

Reply:
There is no file with that name, also all tests are passing

Reply:
Is there a jsfmt.spec.js in that directory (mobile atm)

Reply:
there is no __snapshots__ folder in that directory `tests/css_less_docs`

Reply:
Uh-oh. Raised #3214